app/models/taxon_category.rb

```diff
@@ -0,0 +1,3 @@
+class TaxonCategory < ApplicationRecord
```

Comment:
maybe add some documentation for each model. they are not self explanatory.

Reply:
ok